### PR TITLE
docs: update project list for GSoC 2023

### PR DESCRIPTION
**Pull request by afolarin**
New project list requirements for advertising GSoC 2023 project set.

- [x] archive 2022 project list
- [x] add updated 2023 project list & link
- [x] update project content and mentors

### CODE REVIEW

File: project-ideas_2023/README.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+# RADAR-Base Google Summer of Code (GSoC) 2022
```
Comment (yatharthranjan) [inline project-ideas_2023/README.md]: 2023 ?

**Review by yatharthranjan — COMMENTED** [review]
Looks good. Thanks.

We should probably work on this branch and merge once we have finalised the content and projects (to avoid confusion for prospective gsoc participants)

File: project-ideas_2023/README.md (hunk 1 above)
Comment (afolarin) [reply project-ideas_2023/README.md]: indeed :smile:

**Comment by afolarin** [comment]
I think we can work on this over the next week to 10 days - only merge once people are happy with the content.

File: project-ideas_2023/README.md [hunk 2]
```diff
@@ -6,18 +6,16 @@
...
 
-Questionnaire creation should also support branching logic which is supported by the app.
+### 1A. aRMT Questionnaire Generator
 
-Second stage of this project is to apply the same to generate protocols. Live rendering is not necessary for this. Optionally, if time permits this can be further extend to validate the protocol by generating schedule used on the enrolment-time of a participant.
+**Goals:** 
+The goal of this project is to implement a UI which can be used to generate the questionnaire definition JSON file based on user input and store this generated JSON file to be used by the app. The app should be able to adapt the input type user has chosen and render a sample question that would appear on the app (See wireframe below). We should be able to reuse the components created as part of the questionnaire app to build the live rendering. Questionnaire creation should also support branching logic which is supported by the app.
```
Comment (yatharthranjan) [inline project-ideas_2023/README.md]: Maybe link to an example definitions file would be good to include. Similar for protocol file.

**Comment by Hsankesara** [comment]
Hey @yatharthranjan and @afolarin, it would be great if you could look into the pipeline and model-builder proposal and let me know if it's alright. Thanks.

File: project-ideas_2023/README.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+
+**Dependencies:** Sample/Mock data will be necessary to create the graphs.
+
+**Goal:**
+
+| Goals | Related Issues | Requirements |
+|---------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| General Grafana dashboard configurations | - | - The goal is to create different Grafana dashboard configurations for different data/devices and types of visualisations needed. This would make dashboard setup easier and quicker for different studies. This would also allow easy reuse of configurations across studies.
+
+**Required Skills:** Grafana, TimescaleDB
```
Comment (yatharthranjan) [inline project-ideas_2023/README.md]: @Hsankesara, @mpgxvii  Maybe change this to SQL (as people might not be familiar with timescaledb)?

File: project-ideas_2023/README.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+| Extend the data reader library                     |<br>- Extend data reading from remote server <br>- Add funtionality to read files in various formats such as Avro, json etc |
+| Make feature processing more customizable          | Integrate parameter processing in features.                                                                                                             |
+| Add feature summarization and visualization module | <br>- Add summarization library, which provides summary statistics of features. <br>- Implement support for open source visualization library                       |
+| Develop module of reporting missing data           | Add a module that would report missing data.                                                                                                            |
+| Add industry-wide used features                    | Implement industry-wide features used for mobility, heart rate, phone-interaction etc                                                                   |
```
Comment (yatharthranjan) [inline project-ideas_2023/README.md]: @Hsankesara These are really good, but i think might be overwhelming for participants. Unless you plan to get extension for the work, i would reduce the goals or make some of them optional based on priority.

File: project-ideas_2023/README.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+| Create schemas to send data from Google Activity Recognition API.                                                                 | Refer to [RADAR-Schemas](https://github.com/RADAR-base/RADAR-Schemas), create corresponding schemas and specification to send data from Google Activity Recognition API to RADAR-base platform. |
+| [Create a new plugin](https://github.com/RADAR-base/radar-commons-android#creating-a-plugin) for Google Activity Recognition API. | Retrieve any activity updates and detected activities and send data to RADAR-base server.                                                                                                       |
+
+**Required Skills:** Kotlin, Java
+
+**Difficulty:** Hard
+
+**Expected Size:** 350-hour (Full time)
+
+**Mentors**: [blootvoets](https://github.com/blootsvoets), [peyman-mohtashami](https://github.com/peyman-mohtashami)    ****TBD***
```
Comment (yatharthranjan) [inline project-ideas_2023/README.md]: @zulqarnain343 can you add yourself as a mentor here? Thanks

File: project-ideas_2023/README.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+
+**Current Status**: We ran the pilot study using the model-builder and was able to predict the outcomes in real time. The platform is still in development stage and would require some major refactoring to make it more flexible and easily deployable.
+
+**Source Code**: https://github.com/RADAR-base/model-builder
+
+**Goals:**
+
+| Milestones                                         | Description                                                                                                                                             |
+|----------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Kubernetes Integration         | Intergrating model-builder stack with the [Radar-Kubernetes](https://github.com/radar-base/radar-kubernetes/)                                                           |
```
Comment (yatharthranjan) [inline project-ideas_2023/README.md]: @Hsankesara  i think you can skip the kubernetes one in this as it is quite different from the other tasks and would require good knowledge of k8s, helm, helmfile and the r-base stack. It is also not mentioned in the skills. We can do the integration with radar-k8s fairly quickly as we have done similar ones in the past.

Comment (yatharthranjan) [reply project-ideas_2023/README.md]: You can also add the image from [here](https://docs.google.com/document/d/1vpybkkWRfYF3BLT-Uvemzrjd30knLfT1NMnkOnbbPjI/edit#heading=h.39me92fsq0kj) if you want (not required).

**Pull request merged**